validator/client/beacon-api/genesis.go

```diff
@@ -0,0 +1,70 @@
+//go:build use_beacon_api
+// +build use_beacon_api
+
+package beacon_api
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"net/http"
+	"strconv"
+
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/v3/beacon-chain/rpc/apimiddleware"
+	ethpb "github.com/prysmaticlabs/prysm/v3/proto/prysm/v1alpha1"
+)
+
+func (c *beaconApiValidatorClient) waitForChainStart() (*ethpb.ChainStartResponse, error) {
```

Comment:
pointer receiver for these methods can be omitted since there is no state update